src/main/scala/org/kframework/kore/KoreDSL.scala

```diff
@@ -0,0 +1,58 @@
+package org.kframework.kore
```

Comment:
How about creating a new directory (say `util`) and put this file there? I guess we will add more like this.